apps/web/tsconfig.app.json

```diff
@@ -5,7 +5,7 @@
     "outDir": "../../dist/out-tsc",
     "types": [],
     "target": "ES2022",
-    "useDefineForClassFields": false
+    "useDefineForClassFields": false,
```

Comment:
why you want to have that change already, just to be future proof?

Reply:
i did not add this i think it comes with the latest migration but we take it to be future proof